.github/workflows/conformance-k8s-kind-network-policies.yaml

```diff
@@ -29,7 +29,7 @@ env:
   cilium_cli_ci_version:
   CILIUM_CLI_MODE: helm
   # renovate: datasource=docker depName=kindest/node
```

Comment:
Can we replace with `# renovate: datasource=docker depName=cilium/kindest-node`?